src/github.ts

```diff
@@ -514,14 +514,14 @@ export class GitHub {
     return tags;
   }
 
-  async addLabels(pr: number, labels: string[]) {
+  async addLabels(labels: string[], pr: number) {
```

Comment:
Is this technically a breaking change? It seems like methods default to public in classes.

Reply:
valid point, I should have made methods private at the outset.

Reply:
I went ahead and made this breaking.